### PR TITLE
Get public keys of Datenannahmestellen

### LINKED ISSUE #31: Get all the public keys to which data should be encrypted
Prerequisite of #20

### PR DESCRIPTION
**Pull request by westnordost**
closes #31

### CODE REVIEW

**Comment by westnordost** [comment]
What would be nice if part of the script would actually validate if we got the public keys for all Datenannahmestellen which claim to be capable of decrypting and emitting warnings if they don't

That requires that the kostenträger fetching is combined with the public key fetching, which might actually make sense anyway.

File: README.md [hunk 1]
```diff
@@ -10,6 +10,54 @@ PAID, also known as "Pflegeabrechnung in Deutschland", is a library written in t
...
+import { institutionListReplacer, institutionListReviver } from "kostentraeger/types"
+import { ikPublicKeyInfoReplacer, ikPublicKeyInfoReviver } from "pki/types"
+
+// stringify the data (for persisting it somewhere)
+const pkInfosString = JSON.stringify(publicKeyInfos, ikPublicKeyInfoReplacer)
+const institutionsListString = JSON.stringify(institutionsList, institutionListReplacer)
+
+// revive the data
+const publicKeyInfos2 = JSON.parse(pkInfosString, ikPublicKeyInfoReviver)
+const institutionsList2 = JSON.parse(institutionsListString, institutionListReviver)
```
Comment (michaelkamphausen) [inline README.md:54]: What do think about hiding the replacer / reviver implementation detail? It's a minor change, maybe like this:

```suggestion
import { serializeInstitutionsList, deserializeInstitutionsList } from "kostentraeger/types"
import { serializePublicKeyInfos, deserializePublicKeyInfos } from "pki/types"

// stringify the data (for persisting it somewhere)
const pkInfosString = serializePublicKeyInfos(publicKeyInfos)
const institutionsListString = serializeInstitutionsList(institutionsList)

// revive the data
const publicKeyInfos2 = deserializePublicKeyInfos(pkInfosString)
const institutionsList2 = deserializeInstitutionsList(institutionsListString)
```

**Review by michaelkamphausen — APPROVED** [review]
Looks good!

File: README.md (hunk 1 above)
Comment (westnordost) [reply README.md:54]: Both this data is basically only used in conjunction with each other. So to make the interface more slick here, it should be merged into one data structure anyway, i.e. 

`const institutionData = deserializeInstitutionData(string)`

Regarding the replacer/reviver, I don't mind either way. I just find it a bit weird that the data class would then dictate into which format it is serialized.

Comment (michaelkamphausen) [reply README.md:54]: Good, if the interface could be even simpler, I am in!

Comment (westnordost) [reply README.md:54]: I'd defer that to a separate PR though

**Pull request merged**